docs/xgcpolicy.md

```diff
@@ -104,10 +104,10 @@ The following options can also be specified on the command line with `-Xgcpolicy
 The behavior of the following options is different when specified with `-Xgcpolicy:balanced`:
 
 `-Xcompactgc`
-: Compaction occurs when a System.gc() call is received (default). Compaction always occurs on all other collection types.
+: Compaction occurs when a `System.gc()` call is received (default). Compaction always occurs on all other collection types.
 
 `-Xnocompactgc`
-: Compaction does not occur when a System.gc() call is received. Compaction always occurs on all other collection types.
+: Compaction does not occur when a `System.gc()` call is received. Compaction always occurs on all other collection types.
```

Comment:
Similarly as before, this has nothing to do System GC. It's just disable internal heuristics that trigger Global GC.

We do however two special options that control that for just System GC:

-Xcompactexplicitgc		compact on all system GC's
-Xnocompactexplicitgc	do not compact on system GC's

Reply:
A comment for a section down below that is not edited: I would not bother mentioning -Xpartialcompactgc/  -Xnopartialcompactgc there and actually anywhere in documentation. They are not doing anything even in standard 3 GCs policies

Reply:
As above re -Xnocompactgc. Some help please!
..........
Have removed mention of two deprecated options. (Only referenced in this topic.)

Reply:
Good, but we need to add -Xcompactexplicitgc  and -Xnocompactexplicitgc
It should be the same explanation as for -Xcompactgc and -Xnocompactgc, with emphasis  on _Explicit_ Global GC.

For example, 

for -Xcompactexplicitgc 

Forces compaction in Explicit Global GC cycles (such as those invoked by System.gc(). Compaction in implicit Global GC remains optional, triggered by internal heuristics. 

-Xnocompactexplicitgc 
Disables compaction in Explicit Global GC cycles. Compaction in implicit Global GC remains optional, triggered by internal heuristics.

Reply:
Updates made, and  -X[no]compactexplicitgc added similarly.